### ISSUE TITLE
Update Keep Me Informed Widget

### ISSUE THREAD

**Issue by onaseef** [onaseef/yomobi#26]
Allow users to enter their name and demographic information like city/state and age range.

In the Mobile site builder, we'd have selection lists so users can pick who they want to send a message to, either by region or demographic or name.

See issue #20

**Comment by gilbert** [comment]
I can see the benefit to YoMobi users, but I'm pretty sure mobile site visitors aren't freely willing to enter this information. The conversion rate would be significantly reduced.

**Comment by onaseef** [comment]
I agree that fewer people might fill in all the information. However, numerous customers have asked for it. They've wanted the ability to control who the email or text blasts go out to, and it makes sense.

This is a future feature request.  I'm just logging it for reference.

As part of the builder, we'd let the site owner add the feature and have the option to make the info required of optional.  We could also make it clear to them that the more info the require, the less likely people are to sign up.

The builder would say:  What info would you like to collect:

Name
Email
Phone Number
Age  (Range:  < 18, 18-35, 36-50, 51-65, 66+))
Address: (City, State)
